tests/matchers/Emoji.test.js

```diff
@@ -164,11 +164,13 @@ describe('matchers/Emoji', () => {
   });
 
   describe('onAfterParse', () => {
-    it('returns a single <Emoji/> enlarged', () => {
+    it('returns <Emoji/>-only string enlarged', () => {
```

Comment:
Can you create new tests instead of updating this one?

Should also have a test when there are more content items than the `enlargeUpTo` number.